torch/sparse/__init__.py

```diff
@@ -81,6 +81,8 @@ def mm(mat1: Tensor, mat2: Tensor) -> Tensor:
                values=tensor([ 0.1394, -0.6415, -2.1639,  0.1394, -0.6415, -2.1639]),
                size=(2, 3), nnz=6, layout=torch.sparse_coo)
     """
+    if mat1.is_sparse and mat2.is_sparse:
+        return torch._sparse_matmul(mat1, mat2)
```

Comment:
Is this new dispatch supported in C++, too?

Reply:
I was looking references into the code, here  ./test/cpp and here ./torch/csrc, [there is not much in it about sparse module besides @generated code]. Could you please provide some reference?